EasyCommands/BlockHandlers/BlockHandlers.cs

```diff
@@ -238,6 +245,12 @@ public abstract class TerminalBlockHandler<T> : BlockHandler<T> where T : class,
                 defaultPropertiesByPrimitive[PrimitiveType.VECTOR] = PropertyType.POSITION;
             }
 
+            public override List<T> GetAllBlocksOfType() {
+                List<T> blocks = new List<T>();
+                PROGRAM.GridTerminalSystem.GetBlocksOfType<T>(blocks);
+                return blocks;
+            }
+
             public override List<T> GetBlocksOfType(String name) {
                 List<T> blocks = new List<T>();
                 PROGRAM.GridTerminalSystem.GetBlocksOfType<T>(blocks, block => block.CustomName.Equals(name));
```

Comment:
These two methods are identical with the exception of the predicate passed to `PROGRAM.GridTerminalSystem.GetBlocksOfType<T>(...)`. How about we have the new method call the old one with a null name, and the old method interprets null name in order to not pass a condition? Or, simply pass a condition that returns true to save characters.

```c#
PROGRAM.GridTerminalSystem.GetBlocksOfType<T>(blocks,
    block => name != null
        ? block.CustomName.Equals(name)
        : true);
```

Reply:
Thanks for keeping me honest!  I agree I tried and gave up too early, will address